src/vm/excep.cpp

```diff
@@ -7784,11 +7783,19 @@ LONG WINAPI CLRVectoredExceptionHandler(PEXCEPTION_POINTERS pExceptionInfo)
     //
     return retVal;
 #else // !FEATURE_PAL
+
+#if defined(WIN64EXCEPTIONS) && defined(FEATURE_HIJACK)
```

Comment:
It may be better to remove the #ifndef FEATURE_PAL instead of duplicating the code.